client_cli/pytests/test/accounts/test_accounts_query_filters.py

```diff
@@ -0,0 +1,51 @@
+import json
+import allure
+
+from src.client_cli import iroha, iroha, client_cli
+
+# using existing account to have at least one account in response
+def test_filter_by_domain(GIVEN_new_one_existence_account):
```

Comment:
```suggestion
def test_filter_by_domain(GIVEN_new_one_existing_account):
```
Or am I misunderstanding the convention here?

Reply:
`existence` was used before consistently but i agree that `existing` is more appropriate